inc/azure_c_util/thandle.h

```diff
@@ -54,9 +54,15 @@
 /*the new name is used to define the name of a static function that allocates memory*/
 #define THANDLE_MALLOC_WITH_EXTRA_SIZE(T) MU_C2(T,_MALLOC_WITH_EXTRA_SIZE)
 
-/*given a previous type T, THANDLE_CREATE_FROM_CONTENT introduces a new name for the function that makes of copy of T and returns a THANDLE to the copy*/
+/*given a previous type T, THANDLE_CREATE_FROM_CONTENT introduces a new name for the function that makes of copy of T and returns a THANDLE to the caller*/
 #define THANDLE_CREATE_FROM_CONTENT(T) MU_C2(T,_CREATE_FROM_CONTENT)
 
+/*given a previous type T (usually a struct with a flexible array member), THANDLE_CREATE_FROM_CONTENT_FLEX introduces a new name for the function that makes of copy of T and returns a THANDLE to the caller*/
+#define THANDLE_CREATE_FROM_CONTENT_FLEX(T) MU_C2(T,_CREATE_FROM_CONTENT_FLEX)
+
+/*given a previous type T (usually a struct with a flexible array member), THANDLE_GET_SIZEOF introduces a new name for a user functiont that returns the sizeof of T */
```

Comment:
>functiont [](start = 127, length = 9)

functiont

Reply:
changed to function

---
In reply to: [411489221](https://github.com/Azure/azure-c-util/pull/28#discussion_r411489221) [](ancestors = 411489221)